business_requirement_deliverable/report/br_deliverable_report.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import tools
+from openerp import fields, models
+
+
+class DeliverableReport(models.Model):
+    _name = "deliverable.report"
+    _description = "Deliverable Report"
+    _auto = False
+
+    name = fields.Char('Name', readonly=True)
+    description = fields.Char('Description', readonly=True)
+    partner_id = fields.Many2one('res.partner', 'Customer',
+                                 readonly=True)
+    project_id = fields.Many2one('project.project', 'Master Project',
```

Comment:
@elicoidal Not sure about what we should do

IMO Add dependency to business_requirement_deliverable_project is not an option so we could choose:

1) Dismiss to add `linked_project` field to the report
2) Create the BI report on `business_requirement_deliverable_project` instead of `business_requirement_deliverable`
3) Refactor `business_requirement_deliverable_project` to move field `linked_project` to `business_requirement_deliverable`

cc @darshan-serpent @sudhir-serpentcs